torch/csrc/jit/graph_executor.h

```diff
@@ -50,6 +52,133 @@ struct TORCH_API GraphExecutor {
   std::shared_ptr<GraphExecutorImplBase> pImpl;
 };
 
+struct CaptureList {
```

Comment:
Theres are implementation details of the executor. They should not be exposed.

Reply:
I just want to use DifferentiableGraphOp, which seems to pull all of this in

Reply:
But why do you need that?